addon/components/validated-input/template.hbs

```diff
@@ -39,9 +39,8 @@
   <div class="{{config.css.checkbox}}">
     <label>
       {{one-way-checkbox
-          checked
           name       = name
-          value      = value
+          checked    = (get model name)
```

Comment:
Ah, thanks for explaining the distinction between `value` and `checked` - I never needed that till now :)

I'm not 100% sure if I understand your point completely, and it would certainly help to see an example for the checkboxGroup case you described - could you add that to the dummy app?

To me, it is just important that the simple, single-checkbox use case has the simple API similar to the other components as shown in the demo in this branch. A possible solution might also be to 
bind the model to `checked` by default, but make it overridable when needed.

Thanks in advance for your feedback!